.circleci/config.yml

```diff
@@ -1,23 +1,27 @@
 version: 2
+
+references:
+  circleci_docker_primary: &circleci_docker_primary trussworks/circleci-docker-primary:683f06a3c6b20ba8696a09632c1c26a532df5bf5
+
 jobs:
-  validate:
+  test:
```

Comment:
Does this move us to how other repos have this job named?

This was originally called "validate" because nothing is really getting tested (except some Terraform syntax). It's really just validating the pre-commit hooks were run. As testing goes, it's real weak.

Reply:
Yeah, I copied this from `terraform-aws-default-vpc` because I wanted to ensure I was doing things the same across repos. I agree that `validate` is what its doing, but having different terminology across circleci configs for similar repos can also cause confusion. It just decreases the mental overhead of figuring out why someone chose to do it differently here. 

That being said I'm not married to it.